src/Tickets/Seating/app/blockEditor/update-states-seat-type-by-ticket-id.js

```diff
@@ -0,0 +1,6 @@
+import { storeName } from './store';
+import { dispatch } from '@wordpress/data';
+
+export const useUpdateStatesSeatTypeByTicketId = (clientId) => {
```

Comment:
The `use` part of the name is suggestive of a React hook, since this code is running in the context of a React application this  might be confusing.

I suggest `setSeatTypeForTicket` as an alternative.

But then: one file, for one function with one line of code; does it make sense? I would just call `dispatch(...` directly in the  code. If the code needs to become more complex than a dispatch, then that can be extracted into a function in a separate file and so on.

Reply:
@lucatume i agree it would be better having it in the filter.js since this is a one liner. Regarding tests, i've had issues trying to import anything from the filters.js file in a test file.